packages/components/src/internal/actions.ts

```diff
@@ -96,26 +97,25 @@ export function getGridIdsFromTransactionId(transactionAuditId: number, dataType
 export async function selectGridIdsFromTransactionId(
     gridIdPrefix: string,
     schemaQuery: SchemaQuery,
-    transactionAuditId: number,
+    transactionAuditId: number | string,
     dataType: string,
     actions: Actions
 ): Promise<string[]> {
-    if (!transactionAuditId) {
-        return;
-    }
+    if (!transactionAuditId) return Promise.resolve(undefined);
```

Comment:
Seems like `if (!transactionAuditId) return undefined;` should suffice. Same for next function.